application/app/src/main/java/uk/nhs/digital/uec/api/authentication/localstub/LocalAmazonCognitoIdentityClientStub.java

```diff
@@ -52,7 +54,8 @@ private InitiateAuthResult initiateAuthRequest(String userName) {
   }
 
   private String generateAuthToken(String id, String issuer, String userName, long duration) {
-    Set<String> cognitoGroupNames = new HashSet<>(Arrays.asList(COGNITO_GROUP));
+    Set<String> cognitoGroupNames =
+        new HashSet<>(Arrays.asList(COGNITO_GROUP, ROLE_FUZZY, ROLE_POSTCODE));
```

Comment:
You could do Set.of(COGNITO_GROUP, ROLE_FUZZY, ROLE_POSTCODE) instead of instantiating a hashset

Reply:
I had to rebase this branch off master, this update is probably not related to this PR